### PR TITLE
[Performance] Add support to use SSL certs for load testing.

### LINKED ISSUE #7952: Add support to upload certificates in performance profile.
#### Current Behavior

Right now, there isn’t a support to upload cert files in performance profile, but it's a common test case to send HTTPS request with a self-signed certificate.

#### Desired Behavior

Add support to upload certificates in performance profile.

#### Mockups

---
#### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by MUzairS15**
**Notes for Reviewers**

This PR fixes #7952 

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by MUzairS15** [comment]
One or two items are to be completed before merging.

**Comment by MUzairS15** [comment]
// @leecalcote

**Comment by leecalcote** [comment]
This is for Fortio only, it modifies the performance profile to be able to capture and associate a single file, right?

**Comment by MUzairS15** [comment]
Yes

File: server/handlers/load_test_handler.go [hunk 1]
```diff
@@ -8,6 +8,9 @@ import (
 	"io"
 	"net/http"
 	"net/url"
+	"os"
+
+	// "os"
```
Comment (gyohuangxin) [inline server/handlers/load_test_handler.go:13]: Please remove the redundant comment.

File: server/handlers/load_test_handler.go [hunk 2]
```diff
@@ -166,14 +180,75 @@ func (h *Handler) LoadTestHandler(w http.ResponseWriter, req *http.Request, pref
...
-	// if values have been passed as body we run test using SMP Handler
-	if string(body) != "" {
+	isSSLCertificateProvided := req.URL.Query().Get("cert") == "true"
+
+	/*
+	 When "cert" query param is set the body contains self-signed certs
+	 and not the SMP config, hence we shouldn't use SMP Handler,
+	 if query param is unset/not present presence of body
+	 if values have been passed as body we run test using SMP Handler
+	*/
```
Comment (gyohuangxin) [inline server/handlers/load_test_handler.go:190]: Can we add "cert" as a param in the SMP config?

**Review by gyohuangxin — COMMENTED** [review]
Thanks for adding this feature, I leave some comments.

File: server/handlers/load_test_handler.go (hunk 2 above)
Comment (MUzairS15) [reply server/handlers/load_test_handler.go:190]: Yes, we should add. // https://github.com/service-mesh-performance/service-mesh-performance/issues/389

Comment (gyohuangxin) [reply server/handlers/load_test_handler.go:189]: @MUzairS15 Good!

**Comment by MUzairS15** [comment]
@Abhi `eslint` is failing, how do I fix `make ui-lint` didn't report any error.

**Comment by vishalvivekm** [comment]
@MUzairS15 Let's discuss this on Meshery Dev call. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit#) if you would. :)

**Pull request merged**